### PR TITLE
Missing Date field added 

### LINKED ISSUE #616: Responses does not list "End date field is missing." error message.
When we update an user profile, if the filed notes is removed from the inputs then we see the expected error "End date field is missing." but this not listed under responses.

**To Reproduce**

1 .Go to 'Mentorship relation- send request'.
2. Provide authorization access token.
3. 
```
{
      "mentor_id": "271",
      "mentee_id": "272",
      "notes": "string"
}
```
4. Click on 'Execute'.
5. Scroll down to 'Responses'.
6. 
Screenshot (123)

**Expected behavior**
All the error messages should be listed under the 400 error messages

**Desktop (please complete the following information):**
 - OS: windows 10
 - Browser chrome

### PR DESCRIPTION
**Pull request by Anmollenka**
### Description

End Date field is added.

Fixes #616

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

End field has been added and  can be seen under desciption of error messages.

Screenshot (952)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Anmollenka** [comment]
#616

**Comment by Anmollenka** [comment]
@darecoder please review my pull request.

**Review by SanketDG — DISMISSED** [review]
LGTM

**Comment by darecoder** [comment]
Nice work @Anmollenka.

**Comment by Anmollenka** [comment]
Thank You

File: .env.template [hunk 1]
```diff
@@ -5,4 +5,4 @@ MAIL_DEFAULT_SENDER = <mail-default-sender>
 MAIL_SERVER = <mail-server>
 APP_MAIL_USERNAME = <app-mail-username>
 APP_MAIL_PASSWORD = <app-mail-password>
-MOCK_EMAIL = <True-or-False>
```
Comment (isabelcosta) [inline .env.template]: can you remove this change as this is not related to the issue?

**Comment by Anmollenka** [comment]
Ok @isabelcosta I will make the required changes.

**Comment by Anmollenka** [comment]
@isabelcosta I tried removing that file but when I am trying to recommit it's getting merged with  commits of other people .Can you help me out.Thank You.

**Comment by Anmollenka** [comment]
Shall I create another pull request for this?

**Comment by isabelcosta** [comment]
@Anmollenka this is because you used the `develop` branch for development. Can you search for a bit, how you can update your fork's `develop` branch. That will probably prompt the merge commit, which is fine :)

Let me know if you struggle with it. If you struggle, we can consider creating a new PR, with a different branch. We ask in the CONTRIBUTING.md that you create a new branch before making your changes because this avoids that you run into this kind of problem :)

**Comment by Anmollenka** [comment]
@isabelcosta I have made the required changes.Please review it.Thank You.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
can you please "undelete" the `.env.template` file? you are almost there ;). Then I can approve & merge @Anmollenka

**Comment by Anmollenka** [comment]
@isabelcosta I have removed the `.env.template`  file as it was not required for this issue.Is that okay??
Please review my pull request.

**Comment by vj-codes** [comment]
@devkapilbansal @SanketDG @isabelcosta hey this PR is from develop branch , although it can merged is it a good practice to do so ?

**Comment by devkapilbansal** [comment]
@Anmollenka you should always make a new branch first before creating a pull request. 
@vj-codes I don't think there is any problem if this PR is merged

**Review by isabelcosta — DISMISSED** [review]
Great work @Anmollenka 👏 🎉 
Thank you for your contribution!

**Comment by isabelcosta** [comment]
@Anmollenka can you please fix the merge conflicts, after that ping me and then I can approve again.
cc @devkapilbansal @vj-codes

**Comment by devkapilbansal** [comment]
@Anmollenka some tests are failing. Can you please look into that too

**Comment by Anmollenka** [comment]
> @Anmollenka some tests are failing. Can you please look into that too

Yes,I am working on it.After that I will suash the commits.

File: app/api/resources/mentorship_relation.py [hunk 2]
```diff
@@ -52,6 +52,7 @@ class SendRequest(Resource):
             messages.MENTEE_ALREADY_IN_A_RELATION,
             messages.MENTOR_ID_FIELD_IS_MISSING,
             messages.MENTEE_ID_FIELD_IS_MISSING,
+            messages.END_DATE_FIELD_IS_MISSING,
```
Comment (isabelcosta) [inline app/api/resources/mentorship_relation.py]: I think you have to add an extra `\n%s` in line 42 @Anmollenka :)

**Comment by devkapilbansal** [comment]
@isabelcosta the problem may not be due to these pull requests but due to a bug in the develop branch itself. The same tests are failing in other updated PRs too

**Comment by isabelcosta** [comment]
@Anmollenka can you please update your branch with latest changes from the `develop` branch? These might fix the issues you are having here with tests.
cc @devkapilbansal

**Comment by Anmollenka** [comment]
@devkapilbansal Can you please review my pull request

**Comment by Anmollenka** [comment]
@devkapilbansal Can you please review my pull request.

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1:

**Comment by devkapilbansal** [comment]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

  * _Test1 Description_  End Date Field check
    _Screenshot/gif_:  
test_branch

    _Expected Result_:  `End date field is missing` should be listed in error messages
    _Actual Result_: Same as expected
 
3. Status of PR Changed to: Ready to Merge.

**Comment by devkapilbansal** [comment]
@vj-codes @isabelcosta Can you please re-review this pull request?

**Review by isabelcosta — APPROVED** [review]
Great work @Anmollenka 👏 
Thank you so much for your contribution!

**Pull request merged**